File: DS-Algo-with-Java/Lecture-4/DivByThreeOrFive.java
import java.util.Scanner;

public class DivByThreeOrFive {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Enter a number: ");
            int number = sc.nextInt();

            if (number % 3 == 0 || number % 5 == 0) {

                if (number % 3 == 0 && number % 5 == 0) {
                    System.out.println("The number is divisible by 3 and 5.");
                } else if (number % 3 == 0) {

                    System.out.println("The number is divisible by 3.");
                } else {
                    
                    System.out.println("The number is divisible by 5.");
                }
            }
        }
    }
}

File: DS-Algo-with-Python/Lecture-4/DivByThreeOrFive.py
number = int(input("Enter a number: "))

if (number % 3 == 0) or (number % 5 == 0):
    if (number % 3 == 0) and (number % 5 == 0):
        print("The number is divisible by 3 and 5.")
    elif number % 3 == 0:
        print("The number is divisible by 3.")
    else:
        print("Ther number is divisible by 5.")
